ksqldb-engine/src/main/java/io/confluent/ksql/engine/rewrite/QueryAnonymizer.java

```diff
@@ -310,6 +310,18 @@ public String visitListStreams(final ListStreamsContext context) {
       return stringBuilder.toString();
     }
 
+    @Override
+    public String visitListTables(final SqlBaseParser.ListTablesContext context) {
```

Comment:
Did you check for any other visit methods to override as well?

Reply:
I think that would be good follow-up work. I just happened to notice this one because it showed up in one of my test cases.